test-zone-data/COMMON/arpa

```diff
@@ -45,6 +45,10 @@ $ORIGIN 22.9.19.127.in-addr.arpa.
 $ORIGIN 2.2.0.0.9.0.0.0.9.1.0.0.7.2.1.0.0.0.0.0.3.c.0.0.2.b.0.0.1.a.d.f.ip6.arpa.
 @                       PTR  ns2.zone09.xa.
 
+$ORIGIN zone11.arpa.
+@                       NS   ns1.zone11.xa.
+@                       NS   ns2.zone11.xa.
+
```

Comment:
This is one way to do it. I have moved to creating test case dedicated root, xa and other zones instead to have as few shared files as possible to cut down the risk of conflicts